Консольное Приложение №1

mizyakov-ivan opened the pull request:
Убрана бесконечность в расчетах. Код разнесен в разные классы

mizyakov-ivan committed:
Консольное Приложение №1

mizyakov-ivan committed:
Консольное Приложение №1
Исправлен выход на бесконечность в расчете разделеного счета
Разнесен код по разным классам

faritowich commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,32 @@
+        System.out.println("Сумма которую необхоидмо поделить равна: "+ totalPrice);
+        System.out.println("Количество человек: " + amountPeople);
+        System.out.println("Количество добавленных товаров: "+ amountProducts);
+        System.out.println("Добавленные товары:" + listProducts);
```
Такие подряд идущие println можно объединить в один. Переносы строки можно делать при помощи \n

faritowich commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,32 @@
+    static double totalPrice = 0; // итоговая стоимость товаров
+    static int amountProducts= 0; // количество товаров
+    static String listProducts = ""; // список товаров
+    static Scanner scanner = new Scanner(System.in);
```
Я бы не стал делать программу на статиках. В разработке под андроид используется преимущественно ООП, так что лучше сразу делать таким образом, создавая объекты вручную

mizyakov-ivan committed:
Консольное Приложение №1
Добавлено предупреждение об ошибке, при отрицательной стоимости товара
Убрано дублирование вывода разделенного счета с неправильной формой слова рубль
